### ISSUE TITLE
nhmmer failed to run

### ISSUE THREAD

**Issue by anwarMZ** [tseemann/prokka#398]
Hi, 

I am trying to run prokka on a draft genome for annotation by using the `conda` with version `prokka=1.13.7` installation. Even though it detects `barrnap=0.9` during looking for requirements but still fails to run nhmmer.  see attached screenshot. Any help is appreciated.

![Screenshot 2019-07-27 at 14 03 47](https://user-images.githubusercontent.com/25245737/61994198-d143f780-b077-11e9-8d9f-51bcd78c78ff.png)

**Comment by tseemann** [comment]
Can you just try running `barrnap` alone, with a contigs files ?

**Comment by anwarMZ** [comment]
Its a bit of strange situation. I ran `barrnap` on the genome file in the same directory i was trying prokka before and it stopped after identifying the rRNA but before sorting. To make it more easier for you to read the path which was quite long i just copied the draft genome to another folder few levels up. `barrnap` then worked 👍 , I just ran `prokka` again and that worked too 👍 

My initial guess is that due to my anaconda installation and path, it could not find hmmer and now it did. But i will get in touch again if i see similar problem. I am using `prokka` frequently
